.dockerignore

```diff
@@ -0,0 +1,16 @@
+# Git and GitHub
```

Comment:
I wonder if the easier approach will be to put the Dockerfile in a subdirectory so we don't need all this? The disadvantage here is that we might need two copies of `renv.lock`. Though maybe we could get away with this by making the one in the `docker` directory the "real" one and symlinking from the outer directory to it? I suppose it might depend on how the `renv.lock` file is updated. (You can't link the other way because Docker build won't follow links by design.)

Alternatively, the action could copy that file into the docker directory first? Or maybe just sticking with the .dockerignore is the way to go.

Reply:
What about when someone builds the image locally? Do we write a shell script that does the copying & building and then that’s what gets run via action?

Reply:
Yeah, that was my final worry, and why I went back on my idea at the end. I'm not totally sure how the Docker actions would interact with this, and it seems to me to be worth keeping the "official" Docker build actions rather than trying to roll our own, which would presumably require installing Docker, etc. We could write a shell script to copy and build locally, but I would not use that for the action. So now we have two things to maintain anyway...

Reply:
>  it seems to me to be worth keeping the "official" Docker build actions rather than trying to roll our own

Very good point.